src/components/SearchBy/SearchBy.spec.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SearchBy from '../SearchBy/SearchBy';
+
+
+describe('< SearchBy />', () => {
```

Comment:
I would also add test cases which click buttons and check that props has been called